flexmeasures/ui/templates/crud/accounts.html

```diff
@@ -0,0 +1,38 @@
+{% extends "base.html" %} {% set active_page = "accounts" %} {% block title %} Account listing {% endblock %} {% block divs %}
+
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-sm-2"></div>
+        <div class="col-sm-8">
+            <table class="table table-striped table-responsive paginate">
```

Comment:
Can you add a `<div class="card">` here (and close it later), similar to what we do on the users page? I believe  this should make the styling coherent between these two pages.

Reply:
Followed by a `h3>All accounts</h3>`